Src/Support/Google.Apis.Auth.Tests/OAuth2/GoogleCredentialTests.cs

```diff
@@ -242,6 +242,60 @@ public async Task FromStream_ServiceAccountCredential_GetJwtAccessToken()
             Assert.Equal(3600, payload.ExpirationTimeSeconds - payload.IssuedAtTimeSeconds);
         }
 
+        [Fact]
+        public async Task FromUserCredential_OidcTokenFails()
+        {
+            var stream = new MemoryStream(Encoding.UTF8.GetBytes(DummyUserCredentialFileContents));
+            var credential = GoogleCredential.FromStream(stream);
+            await Assert.ThrowsAsync<InvalidOperationException>(() => 
+            credential.GetOidcTokenAsync(OidcTokenOptions.FromTargetAudience("audience"))).ConfigureAwait(false);
```

Comment:
Nit: indent, possibly even moving the `() =>` onto this line. I don't think you need the `ConfigureAwait(false)` though (anywhere in the tests)